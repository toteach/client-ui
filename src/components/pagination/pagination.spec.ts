import { shallowMount, Wrapper } from '@vue/test-utils';
import Vue from 'vue';

import TPagination from './pagination.vue';

// eslint-disable-next-line max-lines-per-function
describe('TPagination', () => {
  let wrapper: Wrapper<Vue>;
  beforeEach(() => {
    wrapper = shallowMount(TPagination);
  });

  it('is initialized', () => {
    expect(wrapper.name()).toBe('TPagination');
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  it('renders correctly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
