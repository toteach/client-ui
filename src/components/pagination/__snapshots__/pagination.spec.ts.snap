// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TPagination renders correctly 1`] = `
"<div class=\\"t-pagination\\">
  <t-button-stub tag=\\"button\\" type=\\"text\\" nativetype=\\"button\\" class=\\"t-pagination__button\\">

  </t-button-stub>
</div>"
`;
